Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -79,79 +79,90 @@ public final class FormAddressItem: FormValueItem<PostalAddress, AddressStyle>,
                 items.append(item)
             }
         }
-
+        
         delegate?.didUpdateItems(items)
     }
-
+    
     private func create(for field: AddressField, from viewModel: AddressViewModel, subRegions: [Region]?) -> FormItem {
         let item: FormItem
         if let subRegions = subRegions, field == .stateOrProvince {
             item = createPickerItem(from: viewModel, subRegions: subRegions)
         } else {
             item = createTextItem(for: field, from: viewModel)
         }
-
+        
         item.identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: field.rawValue)
         return item
     }
-
+    
     private func createPickerItem(from viewModel: AddressViewModel, subRegions: [Region]) -> FormItem {
         let defaultRegion = subRegions.first { $0.identifier == initialCountry } ?? subRegions[0]
         let item = FormRegionPickerItem(preselectedValue: defaultRegion,
                                         selectableValues: subRegions,
                                         style: style.textField)
         item.title = viewModel.labels[.stateOrProvince].map { localizedString($0, localizationParameters) }
-
+        
         bind(item: item, to: .stateOrProvince)
         return item
     }
-
+    
     private func createTextItem(for field: AddressField, from viewModel: AddressViewModel) -> FormItem {
         let item = FormTextInputItem(style: style.textField)
         item.validationFailureMessage = validationMessage
         item.title = viewModel.labels[field].map { localizedString($0, localizationParameters) }
         item.placeholder = viewModel.placeholder[field].map { localizedString($0, localizationParameters) }
         item.contentType = field.contentType
-
+        
         if viewModel.optionalFields.contains(field) {
             if let title = item.title {
                 item.title = title + " \(optionalMessage)"
             }
         } else {
             item.validator = LengthValidator(minimumLength: 2, maximumLength: 70)
         }
-
+        
         bind(item: item, to: field)
         return item
     }
-
+    
     private func bind(item: FormRegionPickerItem, to field: AddressField) {
         observers[field].map(remove)
         observers[field] = bind(item.publisher, at: \.identifier, to: self, at: \.value.stateOrProvince)
     }
-
+    
     private func bind(item: FormTextInputItem, to field: AddressField) {
         observers[field].map(remove)
-
+        
+        let item = item
         switch field {
         case .street:
             observers[.street] = bind(item.publisher, to: self, at: \.value.street)
+            observe(publisher) { item.value = $0.street ?? "" }
```

Comment:
You probably need another dump of (Item -> View) `observers`.
Current `observers` may become `viewObservers`, and you add another `itemObservers`.

Motivation:
We cleaning `observers[field].map(remove)` here, because based on country different fields can mean different things..